libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/binder/infopanel/InfoPanelBinder.kt

```diff
@@ -124,12 +125,8 @@ internal class MapboxInfoPanelBinder : InfoPanelBinder() {
         layout.findViewById(R.id.infoPanelContent)
 
     override fun applySystemBarsInsets(layout: ViewGroup, insets: Insets) {
-        layout.setPadding(
-            layout.paddingLeft,
-            layout.paddingTop,
-            layout.paddingRight,
-            insets.bottom
-        )
+        layout.updatePadding(bottom = insets.bottom)
```

Comment:
I might be wrong, but we are also applying bottom insets in `InfoPanelComponent`
https://github.com/mapbox/mapbox-navigation-android/pull/6361/files#diff-b90fce80e8d3ff8b253739727929265a5b8a2aa9fb50f5e34bfdcbd8ff8aeb85R34

Reply:
I am aware of this. Thanks. 
This is correct. Binder applies insets faster than `Component.onAttached`. Try removing this and re-runing QA-Test-App. You'll notice that the info panel is laid out with the wrong size at first, then immediately pops to the right size.